gddo-server/assets/templates/pkg.html

```diff
@@ -171,7 +171,7 @@ <h4 class="modal-title">Jump to identifier</h4>
         <div class="panel-heading"><a class="accordion-toggle" data-toggle="collapse" href="#ex-{{.ID}}">Example{{with .Example.Name}} ({{.}}){{end}}</a></div>
         <div id="ex-{{.ID}}" class="panel-collapse collapse"><div class="panel-body">
           {{with .Example.Doc}}<p>{{.|comment}}{{end}}
-          <p>Code:{{if .Example.Play}}<span class="pull-right"><a href="?play={{.ID}}">play</a>&nbsp;</span>{{end}}
+          <p>Code:{{if .ShowPlayLink}}<span class="pull-right"><a href="?play={{.ID}}">play</a>&nbsp;</span>{{end}}
```

Comment:
Then this can contain the decision whether to show the example, like:

```
{{if (and .Example.Play .Standard)}}...{{end}}
```

On second thought, this might not work so great since this template doesn't have access to the top level package, so it's not clear that `.Standard` would mean `.Package.Standard`.